src/services/address.service.ts

```diff
@@ -38,35 +38,98 @@ class AddressService {
     }
 
     public async distance(addressRequest?: any): Promise<any> {
-        // Complete this
+        return new Promise<any>(async (resolve, reject) => {
+            this.request(addressRequest)
+            .then((response) => {
+                if(response.length < 2) {
```

Comment:
We should probably also check the data types to ensure that they're actually valid addresses, too